### PR TITLE
Update single string for onboarding

### PR DESCRIPTION
**Pull request by codemist**
Correcting error with string content.

Removing "Use a custom domain for sharing aliases:" from the step 2 onboarding description.

### CODE REVIEW

**Comment by codemist** [comment]
Just wondering if we have a naming convention for strings that need to be updated but can't be deleted, and can only be resolved by making a new variable.

File: en/app.ftl [hunk 1]
```diff
@@ -500,8 +500,8 @@ multi-part-onboarding-reply-headline = Reply to your emails
 multi-part-onboarding-premium-welcome-description = No more five-alias limit: you can now generate as many custom or random aliases as you need. On desktop, you can use the Relay add-on to create them on the go.
 multi-part-onboarding-premium-domain-description = Using a custom domain, you can create aliases that are easier than ever to remember and share. 
     Need an alias for restaurant reservations? Just say food@yourdomain{ $mozmail } — No need to create the alias beforehand. 
-multi-part-onboarding-premium-get-domain-description = Use a custom domain for sharing aliases: With a custom domain, you can create 
-    aliases without having to generate them beforehand. Need one to sign up for a newsletter? Just say “read@customdomain{ $mozmail }”
```
Comment (codemist) [inline en/app.ftl:504]: This was just added in yesterday, and hasn't been used in the code base.

**Comment by peiying2** [comment]
> Just wondering if we have a naming convention for strings that need to be updated but can't be deleted, and can only be resolved by making a new variable.

1. Replace: When a string is revised, adding `-2` to an existing string ID and then update the string will. This assumes you no longer want the original version.
2. Keep both long term: If you want to  keep both versions, you can keep both the original (same ID and string) and add the revised version and label the ID as `-2` so two versions are related. 
3. Keep both short term: If you want to make original version available until the new version is localized, then you can follow the same approach in #2. You can also add a comment to the original version, like "obsolete string". This way, at some point when most locales have localized the newer version, you want to clean up the file to remove the old version strings, they are clearly marked.

**Review by peiying2 — APPROVED** [review]
@codemist This PR looks good. Let me know if I answered your question and if the question is related to this P:  what you did here is what you intended.

**Comment by codemist** [comment]
Thank you @peiying2! I'll definitely start indicating obsolete strings from now on. Would you say it may be a good practice to set a time maybe one a month or something to prune obsolete strings from the code base?

**Comment by peiying2** [comment]
> Thank you @peiying2! I'll definitely start indicating obsolete strings from now on. Would you say it may be a good practice to set a time maybe one a month or something to prune obsolete strings from the code base?

It's up to the team on how frequent to clean up the code base, but that's a good suggestion.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**